### ISSUE TITLE
 Invalid transaction hash

### ISSUE THREAD

**Issue by Nyrsta** [iotaledger/iri#140]
testnet GUI with lightwallet configuration (VPS)
I happen infrequent this message pop up. 
This for spamming with GUI, sending a value tx, replaying a value tx
(I never have had this message with command line spammer controlled remote from same PC)

java.lang.RuntimeException: Invalid transaction hash
        at com.iota.iri.TransactionValidator.runValidation(TransactionValidator.java:79) ~[iri-1.1.3.10.jar:na]
        at com.iota.iri.TransactionValidator.validate(TransactionValidator.java:85) ~[iri-1.1.3.10.jar:na]
        at com.iota.iri.service.API.broadcastTransactionStatement(API.java:621) [iri-1.1.3.10.jar:na]
        at com.iota.iri.service.API.process(API.java:218) [iri-1.1.3.10.jar:na]
        at com.iota.iri.service.API.processRequest(API.java:147) [iri-1.1.3.10.jar:na]
        at com.iota.iri.service.API.access$000(API.java:81) [iri-1.1.3.10.jar:na]
        at com.iota.iri.service.API$1.handleRequest(API.java:135) [iri-1.1.3.10.jar:na]
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:208) [iri-1.1.3.10.jar:na]
        at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:809) [iri-1.1.3.10.jar:na]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_131]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_131]
        at java.lang.Thread.run(Thread.java:748) [na:1.8.0_131]

**Comment by alon-e** [comment]
TL;DR: this issue is related to GUI wallet.
please open an issue in https://github.com/iotaledger/wallet/issues

the exception in IRI implies that the trytes received to broadcast are invalid.
this means that the hash of the transaction doesn't meet MWM (min. weight mag.) requirements.

this validation is there to make sure you don't send translations which are invalid (they would be dropped my neighbors anyway).
